frontend/src/main/web/src/components/Modal/ModalBody.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { PropTypes } from 'react'
+import { merge } from 'lodash'
+import {
+  Base
+} from '../'
+
+const classes = {
+  base: {
+    p: 'Px(r1) Pt(r1) Pb(r2)',
+    t: 'Ta(s)'
+  },
+  scrollable: {
+    ov: 'Ov(a) Ovx(h)'
+  }
+}
+
+const ModalBody = ({
+  children,
+  scrollable,
+  theme,
+  ...props
+}) => {
+  const themed = merge({},
+    classes,
+    theme
+  )
+  const themedState = merge({},
+    themed.base,
+    scrollable && themed.scrollable
+  )
+  return (
+    <Base {...props}
+      theme={themedState}>
+      {children}
+    </Base>
+  )
+}
+
+ModalBody.propTypes = {
+  children: PropTypes.node,
```

Comment:
Does this allow it to have multiple children, or would they all have to be wrapped in a div or something? Since this component is already wrapping its children in a div, it should allow an array of children too. You can use the following for the default allowed types of children:

```
PropTypes.oneOfType([
  PropTypes.arrayOf(PropTypes.node),
  PropTypes.node]
)
```